### DISCUSSION TITLE
How does Pocketbase "subscription" work?

### DISCUSSION THREAD

**Discussion by jhh78** [pocketbase/pocketbase#5107] [Q&A] [answered]
Greetings.

Thank you for always being so kind to answer.

I would like to know how the "Subscribe" feature of Pocketbase works.
I can only guess that it is probably websocket, gRPC, http.
If you use another method, please let me know.

Thank you.

**Comment by ganigeorgiev** [comment, marked as answer]
The relatime subscriptions are implemented via [Server-Sent Events (SSE)](https://web.dev/articles/eventsource-basics). More information about the realtime APIs you can find in https://pocketbase.io/docs/api-realtime/.
